comp/core/tagger/types/types.go

```diff
@@ -22,6 +22,25 @@ type Entity struct {
 	hash string
 }
 
+// ProductOrigin is the origin of the product that sent the entity.
+type ProductOrigin string
+
+const (
+	// ProductOriginDogStatsD is the ProductOrigin for DogStatsD.
+	ProductOriginDogStatsD ProductOrigin = "dogstatsd"
+	// ProductOriginAPM is the ProductOrigin for APM.
+	ProductOriginAPM ProductOrigin = "apm"
+)
+
+// OriginInfo contains the Origin Detection information.
+type OriginInfo struct {
```

Comment:
Can we have a description of each value these fields can take and their priorities here ?

Reply:
I've added a few description.